### DISCUSSION TITLE
Sites na "unha" ou no elementor?

### DISCUSSION THREAD

**Discussion by evaraujo** [frontendbr/forum#1377] [Perguntas e Respostas]
Boa tarde a todos!

Faço alguns freelance de elaboração de sites para alguns clientes aqui de minha cidade. Sempre fiz os sites "na unha" mesmo, com html, css e javascript/jquery. Ultimamente tenho feito a conversão de sites estáticos para o wordpress, criando temas, serviço bem básico mesmo.

Ando vendo o elementor como uma alternativa mais rápida de elaboração de sites institucionais, que é meu carro chefe, mas sem perder a qualidade.

Quando se trata de sites institucionais sem muitas funcionalidades o elementor pode ser uma alternativa? Eu fico com aquele receio de estar entregando um site "lixo" devido a quantidade de códigos gerada pelo construtor, mas é mais uma questão de ética mesmo.

Na opinião de vocês tem algum problema em fazer sites nestes construtores como trabalho final para os clientes? Vou estar "enganando" o cliente com um site deste tipo?

Desde já agradeço a atenção!

**Comment by tcesarpinho** [comment]
ACF e Custom post da pra fazer tudo.

**Comment by evaraujo** [comment]
Verdade tcesarpinho, não uso o acf nos meus freelas mas no órgão publico que trabalho no portal tem várias partes que utilizam o ACF. Você utiliza o elementor em seus projetos?

Agradeço o comentário!

**Comment by luizbills** [comment]
O maior problema do elementor ao meu ver é quanto mais widgets dele você usa, mais bibliotecas inuteis eles ficam colocando. Recentimente até notei que eles usam 2 framework de slider/carrossel... Além de que para elementos simples que precisaria de 1 tag, o elementor sempre acaba colocando dentro de mais 1 ou 2 tags.

Mas para sites baratos que a performance não é um ponto chave (pois o cliente não está pagando para isso), o elementor ajuda muito.

**Comment by rodineicosta** [comment]
Tenho percebido um crescimento na demanda pela utilização do Elementor, principalmente em comentários no grupo [WordPress Brasil do Telegram](https://t.me/WordPressBrasil). Eu, particularmente, tenho resistência a utilização de '_Page Builder_'. Mesmo tendo feito isso em alguns sites criados a partir do Tema Divi. Minha preferência é de aproveitar o máximo possível de soluções livres criadas pela comunidade. Para sites mais simples, minha sugestão é utilizar o Tema [Coletivo](https://github.com/redelivre/tema-coletivo). Já para sites que necessitem de códigos mais avançados, sugiro o '_Starter Theme_' [Odin](https://github.com/wpbrasil/odin).

Mas, cada um usa o que melhor lhe favorece.

**Comment by evaraujo** [comment]
Show Galera, obrigado pela ajuda! Sou novo no uso do Github portanto não sei como responder em cada tópico, para agradecer pessoalmente cada comentário, mas fica meu agradecimento.

Vou continuar fazendo "na unha" mesmo, na verdade vou dar uma olhada no Tema Coletivo e Odin, os quais nunca ouvi falar, até tentei usar o Elementor, mas apesar da facilidade achei ele bem travado para ajustes finos... algo que no código faria bem mais rápido...

Abraço a todos!

**Comment by nandomoreirame** [comment]
Depende do contexto.

**Comment by rodineicosta** [comment]
@evaraujo, não é possível responder cada comentário. Mas, é possível mencionar o nome de usuário precedido de "**@**".

Sobre o Tema Coletivo e o _Framework_ Odin, são feitos pela comunidade para a comunidade. São livres e todas as informações estão nos referidos repositórios. Qualquer dúvida, envie __Issues__ e responderemos, na medida do possível. ;-)

**Comment by nandomoreirame** [comment]
> @evaraujo, não é possível responder cada comentário. Mas, é possível mencionar o nome de usuário precedido de "**@**".
> 
> Sobre o Tema Coletivo e o _Framework_ Odin, são feitos pela comunidade para a comunidade. São livres e todas as informações estão nos referidos repositórios. Qualquer dúvida, envie **Issues** e responderemos, na medida do possível. ;-)

@rodineicosta é possível sim, clicando nas 3 bolinhas e em "Quote reply"

![image](https://user-images.githubusercontent.com/1318271/53262477-6d2a9b00-36b5-11e9-90f1-f6eb30b05bf7.png)

**Comment by rodineicosta** [comment]
Obrigado pelo esclarecimento @nandomoreirame. Mas, acredito que ele quis dizer sobre respostas aninhadas, como em comentários de blogs e de algumas redes sociais.

**Comment by fernandoRj** [comment]
acho desnecessario e melhor criar o site todo na unha e as partes que vc quer que o cliente edite, use acf, pq geralmente o cliente so vai editar textos e mudar foto msm, o acf cai mt bem pra isso, ou existe outras coisas que um cliente possa alterar no site alem de texto e imagem?

**Comment by allysonsouza** [comment]
Em alguns projetos utilizamos o Elementor, em algumas páginas chave, claro. Por quê? Como o pessoal disse, o cliente na maioria dos casos não irá querer modificar conteúdo nem layout com os recursos de um page builder, porém há contextos em que o uso do Elementor é válido, posso citar:
- O cliente tem um designer com condições de modificar páginas com um page builder conscientemente.
- Você irá cuidar de algum trabalho de alteração de páginas dinâmicas, que podem ser feitas com Elementor e te dar agilidade no trabalho
- Você precisa criar uma landing page totalmente diferente do apresentado pelo tema

E por aí vai, há contextos, basta se planejar. Eis um e-commerce em que utilizamos Elementor na home, pois sempre há alterações, novidades, promoções sazonais a serem criadas, e assim ganhamos muita agilidade no trabalho: [https://comidaboa.delivery/](https://comidaboa.delivery/)
